src/app/visa/c.spec.ts

```diff
@@ -0,0 +1,630 @@
+// write test cases for visa C
+import { calculatePoints, errorMessages } from '@app/domain'
+
+import {
+    academicBackgroundWith,
+    annualSalaryOf,
+    contractingOrganizationOf,
+    japanese,
+    professionalCareerWith,
+    simulationWithCriteriaC,
+    specialOf,
+    universityOf,
+    positionInCompany
+} from '@app/spec.helper'
+
+// non university holder business owner
+// investment banker
+describe('Visa type C point simulation', () => {
+
```

Comment:
Very small nitpick: I'd like to keep spacing between sections consistent, only this line and line 61 have an extra line break 😅